LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2019 Government Digital Service
```

Comment:
`Copyright (c) 2012 Crown Copyright (Government Digital Service)` would be more accurate here, as while I believe GDS has the right to license works under this license, the copyright remains with the Crown. 

I've copied this from https://github.com/alphagov/govuk-puppet/blob/master/LICENSE.txt#L3

Reply:
@cbaines updated, I'd just copied that from another alphagov repo :)

Reply:
Thanks, that looks good now :+1: